run-automation-test.md

```diff
@@ -0,0 +1,185 @@
+ 
+>More information about TeamCity can be found in [this documentation](https://confluence.jetbrains.com/display/TCD10/TeamCity+Documentation).
+## 2. Configure HockeyApp 
+HockeyApp is a service that allows developers to recruit and manage testers, distribute apps, and collect crash reports, among other things.
+
+We need to get the download URL of your app so that Kobiton can install and run it.
+### 2.1 Get HockeyApp API Tokens and App ID
+To get app URL, these two must-have parameters are required : HockeyApp APP ID and API Tokens.
+
+Go to https://rink.hockeyapp.net/manage/dashboard to get App ID and API Tokens to prepare for the next step.
+- **Get App ID**
+
+  Click your app, you can see App ID 
+
+  ![App ID](./assets/img/2-appid.png)
+- **Get API Tokens**
+
+  In the top right corner, click **user icon** -> **My Devices** -> **API Tokens**.
+
+  Create a Hockey App API token for your app with `Full Access` or `Upload and Release` right.
+
+  ![Create App](./assets/img/2-token.png)
+### 2.2. Fetching the app download URL
+
+To get the app download URL, we have provided a script written in NodeJS. After executing that script, you will get download URL of that app, which will be used in the next step.
+
+Downloads the scipt from [this link]('./getAppUrl.js') and copy to your automation test folder.
+
+## 3. Configure automation test script
+### 3.1 Get Username and API key
+Go to https://portal.kobiton.com and login to your Kobiton account.
+- Username
+  + Click *user icon* -> **"Profile"** (you might find it in the top right corner)
+
+  ![get username](./assets/username.png )
+
+- API key 
+  + Click *user icon* -> **"Settings"**
+
+  ![get api key](./assets/api_key.png )
+
+### 3.2 Get desired capabilities
+The desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device.
+  + Click **Devices** in the top navigation bar.
+ 
+    ![click devices](./assets/devices.png )
+ 
+  + Hover over any device you want to test with and click on the Automation settings button (the gear symbol).
+ 
+    ![click auto](./assets/click_auto.png )
+ 
+  + Kobiton automatically translates your desired capabilities into code based on the options you selected. Simply copy and paste this code into your test script under the Input Capabilities section. This will replace the current configuration with the Kobiton device configuration and change the parameters in your test script. 
+  
+  + On the left-hand side, you can select your preferred language, App Type, etc.
+  In this example, we use **NodeJS** as the default language of the script. Therefore, choose **NodeJS** in `Language` section and **Hybrid/Native from Apps** in `App type` section.
+
+  ![auto](./assets/automation.png ) 
+
+### 3.3 Configure automation test script
```

Comment:
If you don't tell them to **use** the samples, at least give them a link to the tutorial on how to write test scripts :)